airflow/models.py

```diff
@@ -771,7 +771,7 @@ class TaskInstance(Base, LoggingMixin):
     end_date = Column(UtcDateTime)
     duration = Column(Float)
     state = Column(String(20))
-    try_number = Column(Integer, default=0)
+    _try_number = Column('try_number', Integer, default=0)
```

Comment:
Is this required? Even if you have a setter?

Reply:
Not sure actually - I thought without this that it would try to write to a column called `_try_number` -- I'm not familiar with SQLA. I'll check.

Reply:
Yes, without this you get this error from SQLA:

> OperationalError: (sqlite3.OperationalError) no such column: task_instance._try_number

Rather than a migration I think it's better to specify the column name.

Reply:
I meant can't we keep it at `try_number = ` instead of `_try_number` :-)?

Reply:
I don't think so, otherwise how do we set the raw value through the setter?

Reply:
@bolkedebruin This is the last comment outstanding. I'm not sure how best to address it. Any ideas?

Reply:
Let’s leave it like this